secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -20,9 +20,15 @@
 from google.cloud import secretmanager, secretmanager_v1
 import pytest
 
+from regional_samples import access_regional_secret_version
```

Comment:
Ditto

Reply:
access_regional_secret_version is package and not a function

Reply:
access_regional_secret_version is a module

Reply:
Oops, you're right! Sorry about that.